src/lib.rs

```diff
@@ -93,8 +93,9 @@ impl<'a> Chip8<'a> {
         chip
     }
 
-    // Decrements timers if not 0
-    fn update_timers(&mut self) {
+    // Decrements timers at 60 Hz if not 0
```

Comment:
Good thing you commented here, I am working off memory and not the docs which is not good. Unit testing this thing will surely end up revealing multiple bugs